TestProjects/iOS-Support/XamarinSidebar/MvvmCross.iOS.Support.Core/App.cs

```diff
@@ -13,7 +15,7 @@ public override void Initialize()
                 .AsInterfaces()
                 .RegisterAsLazySingleton();
 
-            RegisterAppStart(new AppStart());
+            RegisterAppStart(new AppStart(Mvx.Resolve<IMvxNavigationService>()));
```

Comment:
Why not use construct to create the appstart so it injects the navigation service?